docs/studio.rst

```diff
@@ -0,0 +1,94 @@
+Studio
+===============
+
+FEDn Studio is a web-based tool for managing and monitoring federated learning experiments. It provides a user-friendly interface for monitoring the progress of training and visualizing the results.
+
+Getting started
+---------------
+
+Before you can start using Studio, you will need an account. Head over to `studio.scaleoutsystems.com/signup <https://studio.scaleoutsystems.com/signup/>`_  and sign up today!
+
+Create a project
+----------------
+
+Start by creating a new project. A project can be used to organize your work. It can be shared with other users, allowing you to collaborate on experiments.
+
+1. Click on the "New Project" button in the top right corner of the screen.
+2. Continue by clicking the "Create button". The FEDn template contains all the services necessary to start a federation.
+3. Enter the project name (mandatory). The project description is optional.
+4. Click the "Create" button to create the project.
+
+Project overview
+----------------
+
+Once you have created a project, you can find it via the sidebar link Projects. Here you will find the list of all your projects. When inside a project you can see the following tabs in the sidebar:
+
+1. **Dashboard**: The dashboard provides an overview of the project. The controller and combiner(s) are listed under "Network". This is also where you can find the current FEDn version and get the option to upgrade to a newer version if available
+2. **Clients**: management of client configurations and a list of current clients. Observe that this feature does not deploy clients, instead it configures a client config that contains a unique token which is required to connect to the reducer and the combiner
+3. **Combiners**: a list of combiners. Observe number of active clients for each combiner
+4. **Sessions**: a list of sessions with related models. Configure and start a new session. Upload compute package and seed model, set number of rounds, timeout limit etc
+5. **Models**: a list of models generated across sessions
+6. **Events**: a log of events from the combiner and the clients of the federated network
+7. **Settings**: project settings, including the option to delete the project and give access to other users
+
+.. image:: img/studio_project_overview.png
+
+
+Configure and attach clients
```

Comment:
This covers two things, "Initialize the project with a compute package and a seed model", and "connect a client".

Reply:
changed